### ISSUE TITLE
Replace manila-generic with manila-ganesha

### ISSUE THREAD

**Issue by pmatulis** [openstack-charmers/zaza-openstack-tests#702]
In [upgrade_utils.py](https://github.com/openstack-charmers/zaza-openstack-tests/blob/d3ab250984c5766267c5be40425aa8cff43dc372/zaza/openstack/utilities/upgrade_utils.py#L37-L50) I think that `manila-generic` should be replaced with `manila-ganesha`.

**Comment by ajkavanagh** [comment]
I don't think it should be.  In our tests, we still want to check that a subordinate to manila can be upgraded.  manila-generic is a test charm, that is a sub to manila, but that carries very little baggage or configuration with it.  Hence it's quite useful.

There might be an argument that ganesha should be added to the tests, but not as a replacement for manila-generic.
